metals/src/main/scala/scala/meta/internal/metals/ReferenceProvider.scala

```diff
@@ -202,6 +202,7 @@ final class ReferenceProvider(
         scalaPath <- SemanticdbClasspath
           .toScala(workspace, AbsolutePath(path))
           .iterator
+        if scalaPath.exists
```

Comment:
If we removed some files, we would still be looking for references inside them, since the semanticDB files will not be deleted.

Reply:
Can we simply delete this file?

Reply:
That file is managed by Bloop and we shouldn't remove it ourselves.